examples/analytics-iframe-transport-remote-frame.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Requests Frame</title>
+    <script>
+      /**
+       * To receive analytics events in a third-party frame, you must
+       * implement this method, with this signature. Within that method, you
+       * must create a function which processes the analytics events, and
+       * pass that function to the registerAmpAnalytics3pEventsListener() method
+       * of the object which is passed as a parameter to
+       * onNewAmpAnalyticsInstance().
+       * @param ampAnalytics  Call registerAmpAnalyticsEventListener() on this,
+       * passing your function which will receive the analytics events.
+       */
+      window.processAmpAnalyticsEvent = (event, transportId) => {
```

Comment:
thanks for simplifying. but this is a bit different from what I proposed.
i was thinking of exposing the API as `window.context.onAnalyticsEvent(callback)`

1. more consistent with our ampcontext api
2. this gives 3p code to decide when to call the api, as they might be loading something necessary asynchronously.